src/assets/templates/uid_slate_error.html

```diff
@@ -0,0 +1,16 @@
+<div class="cc-page-status cc-page-sorry">
+  <h1 class="cc-visuallyhidden">Problems with CalNet Login</h1>
+  <div class="cc-widget cc-page-status-container">
+    <div class="cc-widget-title">
+      <h2>Unauthorized MAP(Slate) Login</h2>
+    </div>
+
+    <div class="cc-widget-padding">
+      <div class="cc-left">
+        <p>
+          Sorry, CalCentral did not authorize your MAP(Slate) Credentials. You may log out and try again using your CalNet ID and Passphrase. , or contact our <a href="https://sis.berkeley.edu/help/calcentral-support">support team</a>.
```

Comment:
@nkutub - Is this the formatting we're wanting? Or should we remove the period and space before the comma?

> You may log out and try again using your CalNet ID and Passphrase. , or contact our support team.

Reply:
typo, will change